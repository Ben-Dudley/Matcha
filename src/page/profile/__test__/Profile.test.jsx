import React from 'react';
import renderer from 'react-test-renderer';
import Profile from '../index.js';
// import {configure} from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';

// configure({ adapter: new Adapter() });

describe('Profile test', () => {
  test('should snapshot', () => {
    const comp = renderer.create(<Profile />);
    const tree = comp.toJSON();
    expect(tree).toMatchSnapshot();
  });
});