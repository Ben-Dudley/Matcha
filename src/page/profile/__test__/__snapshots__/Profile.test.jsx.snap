// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile test should snapshot 1`] = `
Array [
  <div
    className="profile"
  >
    <div
      className="fio"
    >
      <header
        className="name"
      >
        Серый волк, 25
      </header>
      <p>
        м, гетеро
      </p>
      <p>
        ЮЗАО Москва
      </p>
    </div>
    <img
      alt=""
      className="photo"
      src="volk.jpg"
    />
  </div>,
  <p
    className="info"
  >
    Рейтинг славы: 10000000000000000
  </p>,
  <p
    className="info"
  >
    О себе: в поисках своего зайца
  </p>,
  <p
    className="tags"
  >
    Мультипликация, животные, кулинария
  </p>,
]
`;
